app/src/main/java/com/keylesspalace/tusky/util/MediaUtils.kt

```diff
@@ -0,0 +1,279 @@
+/* Copyright 2017 Andrew Dawson
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>. */
+
+package com.keylesspalace.tusky.util
+
+import android.content.ContentResolver
+import android.content.Context
+import android.database.Cursor
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.graphics.Matrix
+import android.media.MediaMetadataRetriever
+import android.media.ThumbnailUtils
+import android.net.Uri
+import android.provider.OpenableColumns
+import android.support.annotation.NonNull
+import android.support.annotation.Nullable
+import android.support.annotation.Px
+import android.support.media.ExifInterface
+import android.util.Log
+import java.io.*
+
+import java.text.SimpleDateFormat
+import java.util.Calendar
+import java.util.Date
+import java.util.Locale
+
+/**
+ * Class with helper methods for obtaining and resizing media files
+ */
+class MediaUtils {
```

Comment:
In Kotlin you dont need a class if it has only static functions. Just make them top level functions and remove the class, companion object as well as `@JvmStatic`
The functions got some nullability warnings because of the upgrade to api level 28, would be nice to resolve them.

Reply:
Hm, compiler says I still need `@JvmStatic`

Reply:
not if you remove the `object MediaUtils`. Just make them top-level functions.

Reply:
Aha, I wasn't getting the right imports for the autogenerated `MediaUtilsKt` package when I tried before :heavy_check_mark: